addons/mass_mailing/static/tests/mass_mailing_favourite_filter_tests.js

```diff
@@ -407,5 +407,77 @@ QUnit.module('favorite filter widget', (hooks) => {
         assert.isNotVisible(fixture.querySelector('.o_mass_mailing_remove_filter'),
             "should not have option to remove filter because mailing domain is changed");
     });
+
+    QUnit.test('filter widget works in edit and readonly', async (assert) => {
+        assert.expect(4);
+
+        serverData.models.partner = {
+            fields: {
+                name: { string: 'Name', type: 'char', searchable: true },
+            },
+        };
+
+        serverData.models['mailing.filter'].records = [{
+            id: 1,
+            name: 'Azure Partner Only',
+            mailing_domain: "[['name','=', 'Azure Interior']]",
+            mailing_model_id: 2,
+        }];
+
+        serverData.models['mailing.mailing'].records.push({
+            id: 3,
+            display_name: 'Partner Event promotion',
+            subject: 'Early bird discount for Partners!',
+            mailing_model_id: 2,
+            mailing_model_name: 'partner',
+            mailing_filter_count: 1,
+            mailing_filter_domain: "[['name','!=', 'Azure Interior']]",
```

Comment:
Sorry I didn't catch that before but ideally this should match the filter domain ^^
Not a big deal for the test but you know..
```suggestion
            mailing_filter_domain: "[['name','=', 'Azure Interior']]",
```

Reply:
Done 👍🏽 ,  Thanks to you for the review 🙂